pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java

```diff
@@ -169,6 +169,12 @@ private DispatchRate createDispatchRate() {
      * broker-level
      */
     public void updateDispatchRate() {
+        switch (type) {
```

Comment:
Please check all the entrance when this policy updates.
`DispatchRateLimiter#updateDispatchRate(DispatchRate)`
and `DispatchRateLimiter#onPoliciesUpdate` are also used.

Reply:
@Jason918 OK. I have removed the  `DispatchRateLimiter#updateDispatchRate(DispatchRate)` entrance.  `DispatchRateLimiter` will be only from 'DispatchRateLimiter#updateDispatchRate()'. 

For details please see:  'PersistentTopic.java'
*  line [320](https://github.com/apache/pulsar/pull/14038/files#diff-5edf14cc6f25857d0cfdd26b2d3b3141230ecfb0dfa95aebf7583fd76ede4c4bR320)：`DispatchRateLimiter` will invoke  'DispatchRateLimiter#updateDispatchRate()' in the constructor
* line [2429](https://github.com/apache/pulsar/pull/14038/files#diff-5edf14cc6f25857d0cfdd26b2d3b3141230ecfb0dfa95aebf7583fd76ede4c4bR2429)
* line [3027](https://github.com/apache/pulsar/pull/14038/files#diff-5edf14cc6f25857d0cfdd26b2d3b3141230ecfb0dfa95aebf7583fd76ede4c4bR3027)